stl/inc/xutility

```diff
@@ -130,9 +130,13 @@ _NODISCARD constexpr void* _Voidify_iter(_Iter _It) noexcept {
 
 #if _HAS_CXX20
 template <class _Ty, class... _Types,
+#if 0 // TRANSITION, DevCom-XXXX, DevCom-XXXX
     class = void_t<decltype(::new (_STD declval<void*>()) _Ty(_STD declval<_Types>()...))>>
-constexpr _Ty* construct_at(_Ty* const _Location, _Types&&... _Args) noexcept(
-    noexcept(::new (_Voidify_iter(_Location)) _Ty(_STD forward<_Types>(_Args)...))) /* strengthened */ {
+#else // ^^^ no workaround // workaround vvv
+    void_t<decltype(::new (_STD declval<void*>()) _Ty(_STD declval<_Types>()...))>* = nullptr>
+#endif // TRANSITION, DevCom-XXXX, DevCom-XXXX
+    constexpr _Ty* construct_at(_Ty* const _Location, _Types&&... _Args) noexcept(
+        noexcept(::new (_Voidify_iter(_Location)) _Ty(_STD forward<_Types>(_Args)...))) /* strengthened */ {

```

Comment:
If we outdent these, will clang-format simply put them back? Is the situation any better if we duplicate line 132 into both sides of the preprocessor conditional? It's not a hard blocker, but it would be nice if we could return the original indentation of these two lines.

Reply:
Yes, if we duplicate line 132 then clang-format will outdent these.